db/types/custom/json_array.py

```diff
@@ -163,3 +163,37 @@ def unpack(self):
             ArrayLength([param0]),
             param1,
         ])
+
+
+class NotEmpty(DBFunctionPacked):

```

Comment:
I'd like the python class name to reflect that this function is about JSON, similarly to how the id string makes that clear.

Same goes for all function classes that are meant to operate only on JSON values.